magic-link/magic-link-post-type.php

```diff
@@ -100,6 +102,29 @@ public function dt_details_additional_section( $section, $post_type ) {
         }
     }
 
+    public function dt_settings_apps_list( $apps_list ) {
```

Comment:
@kodinkat I'd love for you to add some comments here on what these parameters are used for. And then same for the other magic link type